src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -201,9 +201,17 @@ public class TableMetrics
     /** number of partitions read creating merkle trees */
     public final TableHistogram partitionsValidated;
     /** number of bytes read while doing anticompaction */
-    public final Counter bytesAnticompacted;
+    public final Meter bytesAnticompacted;
     /** number of bytes where the whole sstable was contained in a repairing range so that we only mutated the repair status */
-    public final Counter bytesMutatedAnticompaction;
+    public final Meter bytesMutatedAnticompaction;
+    /** number of bytes that were scanned during preview repair */
+    public final Meter bytesPreviewed;
```

Comment:
Small thing, theres a bit of a naming inconsistency between `bytesPreviewed` / `tokenRangesPreviewed` and `previewedDesynchronizedBytes` / `previewedDesynchronizedTokenRanges`,  maybe we should rename:

`previewedDesynchronizedBytes` / `previewedDesynchronizedTokenRanges`
to
`bytesPreviewedDesynchronized` / `tokenRangesPreviewedDesynchronized`?